### PR TITLE
Fix examples license to CC0-1.0

### PR DESCRIPTION
**Pull request by mwperina**
- Change Golang examples license to CC0-1.0
- Change Ruby examples license to CC0-1.0
- Change Rust examples license to CC0-1.0

### CODE REVIEW

**Comment by mwperina** [comment]
@dougsland As an author of those examples, could you please ack before merging?

**Review by engelmi — APPROVED** [review]
I am approving already, waiting for @dougsland response.

**Pull request merged**
